src/ansys/fluent/core/utils/uploader.py

```diff
@@ -0,0 +1,167 @@
+import os
+from typing import Any, Callable, Optional  # noqa: F401
+
+import ansys.platform.instancemanagement as pypim
+
+
+class _Uploader:
```

Comment:
Since this does upload and download, we can call it something like `FileTransferService` and rename the filename accordingly.

Reply:
Done. Thank you.